src/pages/graphql/schema/checkout/queries/get-payment-order.md

```diff
@@ -0,0 +1,104 @@
+---
+title: getPaymentOrder query
+---
+
+# getPaymentOrder query
+
+The `getPaymentOrder` query retrieves the payment details for the order. You must run the `getPaymentOrder` query  when the Signifyd integration and `hosted_fields` are enabled in [Payment Services](https://experienceleague.adobe.com/docs/commerce-merchant-services/payment-services/payments-checkout/payments-options.html).
+
+## Syntax
+
+```graphql
+{ 
```

Comment:
Wrong syntax. It's

getPaymentOrder(
        cartId: String!
        id: String!
    ) PaymentOrderOutput
}